Gemfile.lock

```diff
@@ -261,14 +261,14 @@ GEM
       rspec-mocks (~> 3.4.0)
       rspec-support (~> 3.4.0)
     rspec-support (3.4.1)
-    rubocop (0.37.2)
-      parser (>= 2.3.0.4, < 3.0)
+    rubocop (0.40.0)
```

Comment:
Not sure changing the rubocop version will change anything, it pulls the rules from .rubocop.yml running rubocop .0.40 vs .0.37 on ManageIQ yield the same results.  Not sure that this will change things, but happy to go back to rubocop 0.73.2